src/gui/progressbardelegate.h

```diff
@@ -29,23 +29,19 @@
 #pragma once
 
 #include <QProgressBar>
-#include <QStyledItemDelegate>
+#include <QStyleOptionProgressBar>
 
-class QStyleOptionProgressBar;
+class QStyleOptionViewItem;
 
-class ProgressBarDelegate : public QStyledItemDelegate
+class ProgressBarDelegate
```

Comment:
It isn't `delegate` now. Just something like "progress bar painter". Please rename it to avoid confusion.